src/pages/index.js

```diff
@@ -104,25 +148,78 @@ function closeModal(modal) {
 
 function handleEditFormSubmit(evt) {
   evt.preventDefault();
-  profileName.textContent = nameInput.value;
-  profileDescription.textContent = descriptionInput.value;
-  closeModal(editModal);
+  setButtonText(submitButton, true, "Saving...");
+
+  api
+    .editUserInfo({
+      name: nameInput.value,
+      about: descriptionInput.value,
+    })
+    .then((user) => {
+      profileName.textContent = user.name;
+      profileDescription.textContent = user.about;
+      editFormElement.reset();
+      resetValidation(
```

Comment:
`resetValidation` is for clearing possible validation errors when you open the **profile** form, because you insert **valid** data into its inputs. 


You don't need it after **submission**


**Please, correct it everywhere in the code**

Reply:
* My remark hasn't been corrected https://github.com/ChrisLaFumee/se_project_spots/pull/1/files#r1985722847